modules/tiles/test/test-utils/compareArrays.ts

```diff
@@ -0,0 +1,14 @@
+const EPSILON = 0.000000001;
+export function areNumberArraysEqual(array1, array2) {
```

Comment:
I feel certain we already have some utility like this, but I am not sure where. I suppose one more cannot hurt...

Reply:
yes. It is a copy&paste. Where we can put such shared code for tests?